public/app/plugins/datasource/jaeger/_importedDependencies/test/helpers/createFetchResponse.ts

```diff
@@ -0,0 +1,15 @@
+import { FetchResponse } from '@grafana/runtime';
```

Comment:
I think this file can be moved out of the importedDependencies as it's really just a helper and it's not that much code so ok to duplicate.